pyproject.toml

```diff
@@ -42,15 +42,13 @@ cryptography = "*"
 kubernetes = "*"
 podman = ">=4.5"
 rq-scheduler = "^0.10"
-# Experimental LDAP Integration https://issues.redhat.com/browse/AAP-16938
-# We are temporarily updating it to use latest from devel branch to keep consistency accross
-# other AAP components
 django-ansible-base = { git = "https://github.com/ansible/django-ansible-base.git", tag = "2024.1.31", extras = [
     "authentication",
     "channel_auth",
 ] }
 jinja2 = "*"
 django-split-settings = "^1.2.0"
+ansible-core = "^2.16.2"
```

Comment:
@Alex-Izquierdo Should this come off

Reply:
The line belongs to devel. Removing or changing that feature should be done in a different PR.